desktop/apps/artist/routes.coffee

```diff
@@ -45,6 +45,7 @@ sd = require('sharify').data
             res.locals.sd.TAB = tab
             res.locals.sd.CURRENT_ITEM = currentItem
             res.locals.sd.ARTIST_PAGE_CTA_ENABLED = !(res.locals.sd.CURRENT_USER? || res.locals.sd.REFERRER?.includes(APP_URL))
+            res.locals.sd.ARTIST_PAGE_GOOGLE_REFERRER = res.locals.sd.REFERRER?.includes('google')
```

Comment:
I'm not super opinionated about naming, but I think `VISITOR_FROM_GOOGLE` or `ARTIST_PAGE_VISITOR_FROM_GOOGLE` may sound like a more self-documenting name.